### ISSUE TITLE
Home + landing pages "Supported Devices" block

### ISSUE THREAD

**Issue by helms-charity** [aemsites/sling#169]
Currently shown on the home page, the logos are selected from a list when authored. See

https://slingtv-b75-stage.adobecqms.net/content/sling-tv/save-this/component-reference-guide.html#supported-devices

![image](https://github.com/user-attachments/assets/058018db-6350-47ad-9b06-37168e013401)

For our version, we have authored a fragment called "supported-devices". I think we should have a sample fragment that contains every logo, which the user can copy, then delete the logos they don't need and save as a new fragment?

**Comment by gayatritulamandi-sling** [comment]
@kmurugulla @helms-charity 

I have completed this issue:

1. Uploaded missing icons under "https://drive.google.com/drive/folders/1gT4GX2V18zwOG3gi0Z1i-orVfJlrH6bw" path
2. Created "home-page-supported-devices" (https://main--sling--aemsites.aem.page/aemedge/fragments/home-page-supported-devices) - Reference of only home supported device list
3. Created "all-supported-devices" fragments (https://main--sling--aemsites.aem.page/aemedge/fragments/all-supported-devices) - Reference of all supported device list
4. Added "home-page-supported-devices" fragment reference path  in "supported-devices" fragment

Final output same as before in home page

![image](https://github.com/user-attachments/assets/b85edf81-11ae-4b3c-9249-1d2b492d20ee)
